package.json

```diff
@@ -45,18 +44,14 @@
     "@babel/preset-env": "7.1.0",
     "@babel/runtime": "7.0.0",
     "babel-loader": "8.0.2",
```

Comment:
Do we keep babel because we plan to use it in the future?

Reply:
It is used in `webpack.config.js`